### ISSUE TITLE
Popup window forced on top (Linux)

### ISSUE THREAD

**Issue by gregorlarson** [eight04/popup-tab#2]
The created popup window is somehow flagged or forced to be above (overlapping) other Firefox windows. It will not lower. I see this on Linux but not Windoz 10. I tried Ubuntu 14 (Unity) and Debian Stretch (Xfce4 ) and problem appears in both.

**Comment by eight04** [comment]
I'm not sure if this is controllable. There is an [`alwaysOnTop` property on Window type](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/windows/Window), but I don't know how to set it.

**Comment by gregorlarson** [comment]
I guess I could try a window-type of panel instead of popup, also, the problem could be coming from the onFocusChanged listener. It might be necessary to check for WINDOW_ID_NONE ref: https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/windows/onFocusChanged

**Comment by jacobmischka** [comment]
It's not the `alwaysOnTop` property, it seems like it's just because of the `popup` type.

![image](https://user-images.githubusercontent.com/3939997/30485137-23160fe2-99f2-11e7-8df6-bc5e62062f52.png)

**Comment by eight04** [comment]
Would `type: "panel"` solve this?

**Comment by jacobmischka** [comment]
No, setting the type to `panel` is actually the exact same as `popup` in this case. I don't know if it can be fixed without an api change for WebExtensions.

**Comment by grahamperrin** [comment]
Also: 

 * https://github.com/aqt/webext-popup/issues/1#issue-262936402

> Currently not possible due to API limitations, …
